blog/electron-36-0.md

```diff
@@ -0,0 +1,131 @@
+  - [New in 135](https://developer.chrome.com/blog/new-in-chrome-135/)
+- Node `22.14.0`
+  - [Node 22.14.0 blog post](https://nodejs.org/en/blog/release/v22.14.0/)
+- V8 `13.5`
+
+Electron 36 upgrades Chromium from `134.0.6998.23` to `136.0.7103.48`, and V8 from `13.5` to `13.6`.
+
+### New Features
+
+- Added `BrowserWindow.isSnapped()` to indicate whether a given window has been arranged via Snap. [#46226](https://github.com/electron/electron/pull/46226)
+- Added `WebContents.focusedFrame` to get the focused frame.
+  - Fixed `WebContents.opener` to specify potential `null` type. [#45667](https://github.com/electron/electron/pull/45667)
+- Added `ffmpeg.dll` to delay load configuration. [#46173](https://github.com/electron/electron/pull/46173) <span style="font-size:small;">(Also in [34](https://github.com/electron/electron/pull/46174), [35](https://github.com/electron/electron/pull/46172))</span>
+- Added `nativeTheme.shouldUseDarkColorsForSystemIntegratedUI` to distinguish system and app theme. [#46598](https://github.com/electron/electron/pull/46598) <span style="font-size:small;">(Also in [35](https://github.com/electron/electron/pull/46599))</span>
+- Added `excludeUrls` to `webRequest` filter and deprecated the use of empty arrays in `urls` property. [#44692](https://github.com/electron/electron/pull/44692) <span style="font-size:small;">(Also in [35](https://github.com/electron/electron/pull/45678))</span>
+- Added support for Autofill, Writing Tools and Services macOS level menu items in context menus via the new `frame` option in `menu.popup`. [#46350](https://github.com/electron/electron/pull/46350)
+- Added support for `system-context-menu` on Linux. [#46399](https://github.com/electron/electron/pull/46399)
+- Improved ASAR integrity checks on Windows. [#46537](https://github.com/electron/electron/pull/46537)
+- Improved performance of `desktopCapturer.getSources` when not requesting thumbnails on macOS. [#46251](https://github.com/electron/electron/pull/46251) <span style="font-size:small;">(Also in [34](https://github.com/electron/electron/pull/46250), [35](https://github.com/electron/electron/pull/46249))</span>
+- Removed 240 FPS limit when use shared texture OSR. [#45669](https://github.com/electron/electron/pull/45669) <span style="font-size:small;">(Also in [35](https://github.com/electron/electron/pull/45781))</span>
+
+### Breaking Changes
+
+### Deprecated: NativeImage.getBitmap()
+
+The `NativeImage.getBitmap()` function is now deprecated and documented as an alias for `NativeImage.toBitmap()`.
+The two functions both return a newly-allocated copy of the bitmap and are functionally equivalent.
+
+### Deprecated: Extension methods and events on `session`
+
+`session.loadExtension`, `session.removeExtension`, `session.getExtension`,
+`session.getAllExtensions`, `extension-loaded` event, `extension-unloaded`
+event, and `extension-ready` events have all moved to the new
+`session.extensions` class.
```

Comment:
Do we have a class that starts with a lower-case letter?

Reply:
based on `session.md` it seems like we referto the actual module as lower case, and an instance of the class as uppercase

Reply:
So it looks like `session.extensions` is an instance property and `Extension` is the class. I'll open a follow-up PR.